package/src/const/chains.ts

```diff
@@ -34,3 +39,65 @@ export const ibcChannels: Record<
     [BridgeType.axelar]: 'channel-208',
   },
 }
+
+export const ics20Channels: PartialRecord<
+  ChainType,
+  {
+    contract: string
+    channels: PartialRecord<ChainType, { origin: string; counterparty: string }>
+  }
+> = {
+  // this is the configuration for CW20 coins with origin terra
+  // it is best to have a single contract for all cw20 tokens
+  [ChainType.terra]: {
+    contract:
+      'terra1e0mrzy8077druuu42vs0hu7ugguade0cj65dgtauyaw4gsl4kv0qtdf2au',
+    channels: {
+      [ChainType.osmosis]: {
+        origin: 'channel-26',
+        counterparty: 'channel-341',
```

Comment:
ampLuna to osmosis, the counterparty channel is needed to getting it back, and this is different from the normal channel.